kubernetes/deployment-production.tmpl

```diff
@@ -36,6 +36,8 @@ spec:
           env:
             - name: ASSET_PREFIX
               value: https://fe-project.zooniverse.org
+            - name: COMMIT_ID
```

Comment:
Won't these be baked into the image from the build process? Are these env variables needed to be set to override the baked in values?

Reply:
I have absolutely no idea.

Reply:
Testing locally, the value at app start seems to be ignored so I think you're right. Mind you, testing locally, I wasn't seeing the bug in #1592 either.